### ISSUE TITLE
Free the blockRenderMap prop.

### ISSUE THREAD

**Issue by Huespal** [jpuri/react-draft-wysiwyg#777]
Hello

I think this lib will be more usable if blockRenderMap prop is allowed to be passed by de developer.

So, in this file: https://github.com/jpuri/react-draft-wysiwyg/blob/master/src/Editor/index.js

make this code:

`
[...]
blockRenderMap={blockRenderMap}
[...]
`
 
to be this code:

`
[...]
blockRenderMap={this.blockRenderMap}
[...]
`

And its consequences.

This allows, in example, to convert the draft-js 'unstyled' types to 'script', or 'div', to obtain lots of new possibilities with the editor. 

Thanks :D

**Comment by neutraali** [comment]
Yeah, currently trying to supply a `blockRenderMap` -prop results in:

`TypeError: Cannot read property 'wrapper' of undefined at DraftEditorContents.render`

**Comment by beny64** [comment]
'wrapper' property is necessary for list items. You should extend the `DefaultDraftBlockRenderMap` (see [doc](https://draftjs.org/docs/advanced-topics-custom-block-render-map#configuring-block-render-map)) e.g.:
```
  myBlockRenderMap = Immutable.Map({ ... });
  extendedBlockRenderMap = DefaultDraftBlockRenderMap.merge(myBlockRenderMap);
```
and then supply `extendedBlockRenderMap` to `blockRenderMap` property.

@Huespal the property `blockRenderMap` is passed in fact. Despite the line
`blockRenderMap={blockRenderMap}`, the next line 
`{...this.editorProps}` does the job (well, the source code is a little bit confusing, but it works - I use 1.12.13).
But keep in mind you override the original `blockRenderMap` imported from [jpuri/draftjs-utils](https://github.com/jpuri/draftjs-utils/blob/eca1508cf74c8b05ce7d3ab39ab28247cb254d5f/js/block.js#L251).

**Comment by quanpham87** [comment]
@beny64 but it's not, because blockRenderMap was not part of the exposed props (from line #505)
